plugins/TagFix_Brand.py

```diff
@@ -85,9 +88,9 @@ def node(self, data, tags):
                                     tags_to_add[tag] = brands_tags[tag]
                             return {"class": 31301, "subclass": 0,
                                     "fix": {"+": tags_to_add}}
-                        else:
-                            return {"class": 31301, "subclass": 1,
-                                    "fix": {"+": {"brand": tags["name"]}}}
+#                        else:
+#                            return {"class": 31301, "subclass": 1,
+#                                    "fix": {"+": {"brand": tags["name"]}}}
```

Comment:
@nlehuby not sure what this part is for. Why just create brand from name when the name is frequent. May be a false positive.

Note: without httis par can also remove the full `_get_frequent_names()`.

Reply:
Note sure to understand :thinking:
This part is actually to create brand from name when name is frequent but not part of nsi.
how do you get frequent names if you remove `_get_frequent_names()`

Reply:
But why do you want to create brand from frequent name, not using nsi? It the cause of #953

Reply:
The purpose was to foster contribution to the nsi: if it's a false positive, a change may be made to remove generic name or add synonyms to existing brand according to the case. If not, the brand metadata may be missing in nsi (and adding a `brand` tag in OSM is still a good addition).

But if you think there are too much false positive, you can remove it and only use nsi and add brand tags to already known brands objects.

Reply:
Ok thank you, Yes I prefer remove possible false positive.